src/fides/api/ops/schemas/messaging/messaging.py

```diff
@@ -216,8 +236,19 @@ class MessagingServiceSecrets(Enum):
     TWILIO_API_KEY = "twilio_api_key"
 
 
+class MessagingServiceSecretsMailchimpTransactional(BaseModel):
```

Comment:
missed this on the review - and it's not _strictly_ necessary it seems - but we should probably add this class to the `SUPPORTED_MESSAGING_SERVICE_SECRETS` constant on L370 that's used for type hints